lib/verk/stats.ex

```diff
@@ -12,6 +12,17 @@ defmodule Verk.Stats do
     %{processed: to_int(processed), failed: to_int(failed)}
   end
 
+  @doc """
+  Total amount of processed and failed jobs per queue
+  """
+  @spec queue_totals(GenServer.server) :: Map.t
+  def queue_totals(redis \\ Verk.Redis) do
+    for {queue, _} <- Application.get_env(:verk, :queues), into: %{} do
```

Comment:
There's just one problem with this approach. One can add/remove queues dynamically so the application env is not enough to give this information. 

Why don't we simplify for now receive the queue name as part of the `queue_totals` arguments and call from here:

https://github.com/edgurgel/verk_web/blob/436806978af3d37dad7ba10218b4bb084241f71b/web/views/page_view.ex#L4-L7

What do you think?